main.js

```diff
@@ -14,11 +14,24 @@ var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probat
 async function acceptStudent(id, fullName, age, address){
     let attributes = { ID:id, Name: fullName, Age: age, Address: address };
     await db.put(id, attributes);
-    return await db.get(id, function(err, value){
+    await db.get(id, function(err, value){
         if(err){
             console.log(err);
         } else{
             console.log(value, status[0]);
         }
     })
+    var scheduleDate = new Date('March 1, 2021 08:30:00');
+    return await scheduleInterview(id, scheduleDate);
```

Comment:
removing this lines, move it after line 11

Reply:
the lines are moved sir but we have to return the value of id variable in the acceptStudent function.